paper/main.tex

```diff
@@ -1145,7 +1132,7 @@ \subsubsection*{Synthetic data: simulating dynamic first-order correlations}
   
   \paragraph*{Ramping.}  We generated a third set of 100 synthetic
   datasets whose underlying correlations changed gradually over time.
-  For each dataset, we constructed two \textit{anchor} covariance
+  For each dataset, we constructed two anchor covariance
```

Comment:
Change to:

For each dataset, we constructed two ``anchor'' covariance

Reply:
done